backend/resolver/aws/aws.go

```diff
@@ -168,10 +168,15 @@ func (r *res) Search(ctx context.Context, typeURL, query string, limit uint32) (
 		if err != nil {
 			return nil, err
 		}
+
 		if ok {
-			return r.instanceResults(ctx, patternValues["account"], patternValues["region"], []string{patternValues["instance_id"]}, limit)
+			// in the case of account not found, we continue to the below flow of only being concerned with the ID, rather than the account and region
```

Comment:
this is different behavior though, right? should we strip off `aws://` instead somewhere?

Reply:
There was an attempt to do that earlier using `TrimPrefix` but it was decided that that is not a good solution as per Mike's comment:

> I dont think hard coding here is the way to go just for kubectl output, we should think of a different way, maybe something thats configurable. Or just got back to the drawing board rethink this whole issue.

Valid choices for accounts right now are things like `zimride` , `zimridestaging` etc. Without this PR, if the user enters an `aws:///` input, they would just get an error of `account not found`. This PR allows for them to not just get stuck at that error, but be able to continue using the default fan-out search. Note that a while ago before I added the error check PR (https://github.com/lyft/clutch/pull/2041) if you entered an `aws:///` input it would just panic.

Reply:
I just wonder if we shouldn't internalize that to our implementation somehow, either by passing a strip function into the frontend (i think this is easiest maybe?) or extending the backend. i.e. the configurable option.

IMO, it would be strange for users to request `badprefix://account000/foo` only for the resolver to return `account999/foo`

Reply:
lets sync on this tomorrow (thursday)

Reply:
update: we synced and the plan is to have a configureable function that we can pass via the config to manipulate strings (takes in a string, returns a string) to be able to operate on resolvers